tethysext/atcore/public/css/center.css

```diff
@@ -6,7 +6,7 @@
 .center-parent:before {
   content: '';
   display: inline-block;
-  height: 100%;
+  height: 50%;
```

Comment:
Why was this change needed? This should be 100%.

Reply:
At first I couldn't remember why we changed this, but this is what displays the loading bar when we add a resource. We have this ::before so that we can center the loading bar. If the height is 100% you can't see the bar, just a gray screen. 50% for the ::before makes it so the bar shows right after, near the vertical center.